scripts/check_stats.js

```diff
@@ -0,0 +1,41 @@
+const config = require('pelias-config').generate();
+const es = require('elasticsearch');
+const client = new es.Client(config.esclient);
+
+async function stats() {
+    try {
+        const response = await client.search({
+            index: config.api.indexName,
```

Comment:
I think this should be `config.schema.indexName`?

- https://github.com/pelias/schema/blob/master/scripts/create_index.js#L29
- https://github.com/pelias/config/blob/master/config/defaults.json#L73

Reply:
I wasn't sure about this. In my opinion you want to get the stats from your active queried index. But I guess you can argue for both sides.

Reply:
This documentation is actually wrong, I'll fix it:

https://github.com/pelias/api/blob/6037d952d2ed652f883803cf9a590fe3ad968173/README.md?plain=1#L71

The API reads from `config.schema.indexName` too, so using the same variable will guarantee that they will point at the same index.

Reply:
I see the confusion now, we have two variables for essentially the same thing.
@arnesetzer is there a valid scenario where these two variables might differ?

Reply:
"Updating" the elasticsearch database. We use the old index (`api.indexName`) for requests while importing data to the new index (`schema.indexName`). After the importing processes are finished we just need to swap the index and you have almost no downtime.

Reply:
I recommend [aliases](https://www.elastic.co/guide/en/elasticsearch/reference/current/aliases.html) for this scenario, we used it in the past with great success.

basically it acts like a symlink to an index so you can switch `pelias->pelias-2025-01` to  `pelias->pelias-2025-02` with zero downtime by sending a command to Elasticsearch.

Reply:
sorry, I need to chat with Julian about this some more before we merge it, I wasn't totally up-to-date on the fact we had two variables and the purpose of that.